### ISSUE TITLE
Unregistered content hosts crashes katprep_snapshot

### ISSUE THREAD

**Issue by stdevel** [stdevel/katprep#52]
When checking unregistered content hosts, `katprep_snapshot` crashes:
```
INFO:katprep_snapshot:Checking system 'pinkepank.test.loc' (#123)...
...
TypeError: 'NoneType' object has no attribute '__getitem__'
```
Environment:
- Red Hat Enterprise Linux 7.4 (x86_64)
- Red Hat Satellite 6.2.14

**Comment by stdevel** [comment]
Turns out, that the dictionary `["content_facet_attributes"]["errata_counts"]` is `null`. Usually it should contain stuff like this:
```
{u'security': 0, u'bugfix': 0, u'enhancement': 0, u'total': 0}
```
Testing in branch `bug-unreg-chost_issue52`.

**Comment by stdevel** [comment]
Also saw, that Red Hat Satellite 6.2.x still does not offer the `ip6` key - so making it more failure-resistant is key..

**Comment by stdevel** [comment]
Fixed in commit b83b965033a8bbe15a03a2e7035122d39954a558.
